nav2_rviz_plugins/src/navigation_dialog.cpp

```diff
@@ -140,4 +146,5 @@ NavigationDialog::startNavigation(double x, double y, double theta, std::string
   }
 
   timer_.start(100, this);
-}
+
+}
```

Comment:
I'd get rid of the blank line before the brace. This might cause a linter failure and is unneeded. And while you're down here, please add a carriage return after the brace. Some tools throw a warning if there is a character on the last line.

Reply:
fixed it.